### PR TITLE
Adding ability for mods to permanently delete avatars

### PR DESCRIPTION
**Pull request by kev306**

### CODE REVIEW

**Comment by kev306** [comment]
TODO: update swal, change button to Delete permanently. Move rules above the avatar image

File: src/routes/profile/index.tsx [hunk 1]
```diff
@@ -157,6 +161,66 @@ router.post(
   },
 );
 
+router.post('/mod/deleteuseravatar', isModMiddleware, async (req, res) => {
```
Comment (vck3000) [inline src/routes/profile/index.tsx:164]:
```suggestion
router.post('/mod/deleteuseravatar', isModMiddleware, async (req: EnrichedRequest, res) => {
```

File: src/views/components/avatar/avatarLookup/avatarLookup.tsx [hunk 2]
```diff
@@ -87,25 +91,126 @@ export function AvatarLookup() {
...
+        <div class="avatarSet" style="display: flex; justify-content: center; align-items: center;">
+          <img src="${lastSelectedAvatarSet.resLink}" alt="test" class="avatarImg">
+          <img src="${lastSelectedAvatarSet.spyLink}" alt="test" class="avatarImg">
+        </div>
+        <div style="text-align: left; font-size: 16px">
+          <p>Possible reasons include:</p>
+          <ul>
+            <li>Accidental avatar approval</li>
+            <li>Removing duplicate avatars</li>
+            <li>Removing a temporarily set avatar (e.g. Tourney avatars)</li>
```
Comment (vck3000) [inline src/views/components/avatar/avatarLookup/avatarLookup.tsx:151]: Let's rm this

**Review by vck3000 — APPROVED** [review]
🛳️

File: src/routes/profile/index.tsx [hunk 3]
```diff
@@ -171,9 +171,9 @@ router.post('/mod/deleteuseravatar', isModMiddleware, async (req, res) => {
   }
 
   const modWhoProcessed = req.user;
```
Comment (vck3000) [inline src/routes/profile/index.tsx:173]: IUser

**Pull request merged**